assets/Scripts/Managers/LoggingManager.cs

```diff
@@ -189,7 +236,45 @@ public class LoggingManager : MonoBehaviour {
 		{
 			writer.WriteLine(currentEntry);
 		}
-	}
+
+		string b = System.Enum.GetName(typeof(GameType), _gameType);         //To Get the name of the enumerator
+
+		logs["Email"].Add("hello@email.test");
+		logs["Date"].Add(date);
+		logs["Time"].Add(time);
+		logs["UserID"].Add(_userID.ToString());
+		logs["GameType"].Add(System.Enum.GetName(typeof(GameType), _gameType));
+		logs["InputType"].Add(System.Enum.GetName(typeof(InputType), _inputType));
+		logs["InputResponders"].Add(System.Enum.GetName(typeof(InputResponders), _inputResponders));
+		logs["HitType"].Add(_hitType);
+		logs["TargetNumber"].Add(_targetNumber.ToString());
+		logs["TargetID"].Add(_targetID.ToString());
+		logs["SessionTime"].Add(_sessionTime.ToString());
+		logs["DeltaTime"].Add(_deltaTime.ToString());
+		logs["TargetX"].Add(_targetPos.x.ToString());
+		logs["TargetY"].Add(_targetPos.y.ToString());
+		logs["HitX"].Add(_hitPos.x.ToString());
+		logs["HitY"].Add(_hitPos.y.ToString());
+		logs["HitOffsetX"].Add(_hitPos.x.ToString());
+		logs["HitOffsetY"].Add(_hitPos.y.ToString());
+		logs["OutsetTargetX"].Add(_outsetTarget.x.ToString());
+		logs["OutsetTargetY"].Add(_outsetTarget.y.ToString());
+		logs["TargetDeltaX"].Add(_targetPos.x.ToString());
+		logs["TargetDeltaY"].Add(_targetPos.y.ToString());
+		logs["OutsetHitX"].Add(_outsetHit.x.ToString());
+		logs["OutsetHitY"].Add(_outsetHit.y.ToString());
+		logs["DeltaHitX"].Add(_hitPos.x.ToString());
+		logs["DeltaHitY"].Add(_hitPos.y.ToString());
+		logs["TargetDiameter"].Add(_diameter.ToString());
+		logs["ColliderDiameter"].Add(_diameter.ToString());
+		logs["Backtracking"].Add(_backtracking.ToString());
+		logs["ErrorTargetID"].Add(_errorTargetID.ToString());
+		
+
+		connectToMySQL.AddToUploadQueue(logs);
+		connectToMySQL.UploadNow();
```

Comment:
This code might be called many times and in that case, we are making a lot of upload calls. Here is my suggestion:

- Keep appending new logs to the existing Dictionary, using your `logs[""].Add()` calls.
- Once the test is done, then you call the `connectToMySQL.AddToUploadQueue()` and `connectToMySQL.UploadNow()` functions.

This way, the upload call only happens 1 time, which minimizes traffic. 💾

Reply:
It sounds logical. Now it's done 👍